Gemfile.lock

```diff
@@ -1,36 +1,32 @@
 GIT
```

Comment:
I don't even think we should have `Gemfile.lock` checked in - can you just delete this @ranjib?